src/resources/MachineLearning/RGAConfiguration/RGAConfigurationInterface.ts

```diff
@@ -0,0 +1,54 @@
+import {FilterConditions} from '../FilterConditions.js';
+import {DocumentRequirementStatus} from '../Document.js';
+
+export interface RGAPreviewParams {
+    /**
+     * The sources to consider.
+     */
+    sources: string[];
+    /**
+     * An array of filtering conditions.
+     */
+    filterConditions: FilterConditions[];
```

Comment:
This is optional (I was able to call preview in swagger without including filterConditions)

Reply:
Yes you are right! I changed all these to be optional as consistent with the Semantic PR